docs/excel/excel-add-ins-worksheets.md

```diff
@@ -274,6 +274,30 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+## Detect data changes
+
+Your add-in may need to react to users changing the data in a range. To detect these changes, [Worksheets](/javascript/api/excel/excel.worksheet) provide an `onChanged` event. This event fires whenever the data is changed in a particular worksheet. Event handlers for `onChanged` receive a [WorksheetChangedEventArgs](/javascript/api/excel/excel.worksheetchangedeventargs).
```

Comment:
First sentence should end with "worksheet" (not "range").

Then,suggest that you replace the 3 sentences after the opening sentence with the following:

To detect these changes, you can [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the `onChanged` event of a [worksheet](/javascript/api/excel/excel.worksheet). Event handlers for the `onChanged` event receive a [WorksheetChangedEventArgs](/javascript/api/excel/excel.worksheetchangedeventargs) object when the event fires.

(You can edit this comment in GitHub to copy/paste this markdown if you'd like to use it.)